test/data/Mock/RucioMockData.json

```diff
@@ -1392,6 +1392,19 @@
   "scope": "cms",
   "type": "CONTAINER"
  },
+ "listDataRules:[('account', 'ms-pileup'), ('scope', 'cms')]": [{
+  "state": "OK",
+  "id": "123",
+  "rse_expression": "T2_XX_CERN",
+  "locks_ok_cnt": 2,
+  "locks_replicating_cnt": 2,
+  "locks_stuck_cnt": 2,
+  "scope": "cms",
+  "name": "/MinimumBias/ComissioningHI-v1/RAW",
+  "account": "ms-pileup"
+ }],
+ "evaluateRSEExpression": ["T2_XX_CERN"],
```

Comment:
I am not sure these 2 lines (this and the one below) are actually doing what they are supposed to. It looks like you missed the call signature here.

Reply:
According to my unit tests output it is correct and these mock-up functions are called.